app/client/index.tsx

```diff
@@ -10,8 +10,20 @@ import { Watch } from "./components/Watch";
 import { Hist } from './components/Hist';
 import { UserOverview } from './components/Me';
 import startupHook from "./ts/startup-hooks";
+import Config from "./ts/global-config";
+import { Maybe } from "./ts/maybe";
 
-const sidebarRoutes = [
+const CONFIG: Config = Config.getInstance();
+
+interface SidebarRoute {
+    title: string;
+    path: string;
+    exact: boolean;
+    component: any,
```

Comment:
This makes me 😢 but to declare it of type `React.Component` gets super messy with the type parameters, so right here the type safety is not worth it.